windows/src/engine/inst/build.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../../../../resources/build/builder.inc.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+source "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+builder_describe "Installation files for Keyman Engine for Windows" \
+  clean configure build test publish
+
+# NOTE: not using deps here because we will only do this in the 'publish' phase
+# after all other builds complete
+
+builder_describe_outputs \
+  publish       /windows/src/desktop/inst/keymanengine.msm
```

Comment:
This should either be 
`$KEYMAN_ROOT/windows/release/${VERSION}/keymanengine.msm`
of the local version before copy_installer
`keymanengine.msm`
Either case nothing is copying it to where this line suggests.
Also should see comment below

Reply:
Good catch. Fixed.